src/onegov/event/collections/events.py

```diff
@@ -261,7 +270,7 @@ def from_import(
                 Event.meta['source'] == event.meta['source']
             ).first()
 
-            if purge:
+            if to_purge:
                 purged -= {event.source}
 
             if existing:
```

Comment:
This is a little much commit noise here... It seems that the lines from here up until `tags = [tags]` didn't change, only in formatting. It's always good to keep the diff as small as possible